src/Checkbox/Checkbox.styles.ts

```diff
@@ -0,0 +1,33 @@
+import styled from 'styled-components';
+import { CheckboxProps } from './types';
+import { getBackgroundColorFromState, getBorderColorFromState, getColorFromState } from './utils';
+
+interface StyledCheckboxProps extends CheckboxProps {
+  $indeterminate?: boolean;
+}
+
+export const StyledCheckbox = styled.input<StyledCheckboxProps>`
+  appearance: none;
+  border-radius: 2px;
+  border: 1px solid ${getBorderColorFromState};
+  cursor: pointer;
+  flex: none;
```

Comment:
Is this necessary?, the display value is `inline-block` so flex values won't apply.

Reply:
Correct, in the first implementation Checkbox had label and this property was for that. Removing it.